src/client/debug.go

```diff
@@ -2,31 +2,16 @@ package client
 
 import (
 	"io"
-	"time"
 
-	"github.com/gogo/protobuf/types"
 	"github.com/pachyderm/pachyderm/src/client/debug"
 	"github.com/pachyderm/pachyderm/src/client/pkg/grpcutil"
 )
 
-// Dump writes debug information from the server to w.
-func (c APIClient) Dump(w io.Writer) error {
-	goroClient, err := c.DebugClient.Dump(c.Ctx(), &debug.DumpRequest{})
-	if err != nil {
-		return grpcutil.ScrubGRPC(err)
-	}
-	return grpcutil.ScrubGRPC(grpcutil.WriteFromStreamingBytesClient(goroClient, w))
-}
-
 // Profile writes a pprof profile for pachd to w.
-func (c APIClient) Profile(profile string, duration time.Duration, w io.Writer) error {
-	var d *types.Duration
-	if duration != 0 {
-		d = types.DurationProto(duration)
-	}
+func (c APIClient) Profile(profile *debug.Profile, worker *debug.Worker, w io.Writer) error {
 	profileClient, err := c.DebugClient.Profile(c.Ctx(), &debug.ProfileRequest{
-		Profile:  profile,
-		Duration: d,
```

Comment:
Are we removing the Duration argument here? It seems like we still want some way to be able to specify how long to take the memory profile for, although maybe we can just say that a minute is always a reasonable amount of time.